CONTRIBUTING.md

```diff
@@ -19,6 +19,16 @@ This creates an isolated virtual Python environment ("virtualenv") just for
 use with Sygnal, then installs Sygnal along with its dependencies, and lastly
 installs a handful of useful tools
 
+If you get `ConnectTimeoutError`, this is caused by slow internet whereby `pip` has a 
+default time out of _15 sec_. Run the following command by specifying timeout to be more
+than 15 sec.
```

Comment:
Instead of repeating the code block, maybe just add the timeout note in line here?

E.g., instead of saying "Run the following command by specifying timeout to be more than 15 sec." you could say "You can specify a larger timeout by passing `--timeout 120` to the `pip install` command above."

Reply:
got it, Thank you